interface/main/messages/messages.php

```diff
@@ -127,7 +127,16 @@
         $noteid = $_POST['noteid'];
         $form_note_type = $_POST['form_note_type'];
         $form_message_status = $_POST['form_message_status'];
-        $reply_to = $_POST['reply_to'];
+
+        //Need to do this because sometimes there will be multiple patients separated by semicolon, sometimes 1 patient with semicolon, and sometimes 1 patient without semicolon
+        if (strpos($_POST['reply_to'], ';') !== false){
```

Comment:
Isn't this only supposed to be true if the semicolon is at the end?

Reply:
true, changed it to `rtrim`